### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AOstenfeld**
# Pull Request
## Description
Update README for '"pip install turbopy"

This pull request addresses #132 

## Checklist
The following items have been checked for this PR:

- [x] Conforms to PEP8 style standards (check with `pylint`, `flake8`, or similar)
- [x] Code is documented with docstrings, or docstrings have been updated as appropriate
- [x] New unit test/integration test have been added to check new code
- [x] All existing tests still pass

### CODE REVIEW

**Comment by AOstenfeld** [comment]
Is this sort of what you were thinking @arichar6 ?

**Comment by arichar6** [comment]
I was thinking of a bigger change. There are currently two sections "turboPy Conda environment" and "turboPy development environment". I think instead it should be something like "Installing turboPy" and "turboPy development environment". The stuff in the (old) section "turboPy Conda env" should go in the new "dev env" section, and the new "installing" sections should have the `pip install turbopy` command (with more description)

**Comment by AOstenfeld** [comment]
oh ok

**Comment by AOstenfeld** [comment]
How does the `pip install turbopy` command work? Do you have to install the conda environment first or do you just run the command? I haven't installed it that way before.

**Comment by padamson** [comment]
There is no need to explain how to install or use pip. It is a well known method. You can install it with `conda install pip` into your base or any conda environment.

File: README.md [hunk 1]
```diff
@@ -37,10 +43,6 @@ turboPy Conda environment
 	- If you just plan to develop a code using the existing turboPy framework: `pip install .`
```
Comment (arichar6) [inline README.md:43]: I don't think we need this. If they want to just develop an app, then they should just use `pip install turbopy`
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ turboPy Conda environment
 -   Activate: `conda activate turbopy`
 -   Install turboPy into the environment (from the main folder where setup.py is): 
 	- Install turboPy in editable mode (i.e. setuptools "develop mode") if you are modifying turboPy itself: `pip install -e .`
```
Comment (arichar6) [inline README.md]:
```suggestion
-   Install turboPy into the environment in editable mode (from the main folder where setup.py is): `pip install -e .` 
```

File: README.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ turboPy Conda environment
 -   Activate: `conda activate turbopy`
 -   Install turboPy into the environment (from the main folder where setup.py is): 
 	- Install turboPy in editable mode (i.e. setuptools "develop mode") if you are modifying turboPy itself: `pip install -e .` 
-	- If you just plan to develop a code using the existing turboPy framework: `pip install .` 
+	- If you just plan to develop a code using the existing turboPy framework: `pip install turbopy`
```
Comment (arichar6) [inline README.md]: If they are just using turboPy, then they should `pip install turbopy`
```suggestion
```

Comment (arichar6) [reply README.md]: oops... made this comment above already

**Pull request merged**